script.js

```diff
@@ -0,0 +1,150 @@
+const mobileOpen = document.getElementById('open-menu');
+const mobileShow = document.getElementById('navbar-item-mobile');
+const mobileClose = document.getElementById('close-menu');
+const popupOpen = document.querySelectorAll('.card-btn');
+const inputName = document.getElementById('user_name');
+const inputEmail = document.getElementById('user_email');
+const sumbitMessage = document.getElementById('sumbit-action');
+const errorMessage = document.getElementById('error');
+
+mobileOpen.addEventListener('click', () => {
+  if (mobileShow.style.display === 'block') {
+    mobileShow.style.display = 'none';
+  } else {
+    mobileShow.style.display = 'block';
+  }
+});
+
+mobileClose.addEventListener('click', () => {
+  if (mobileShow.style.display === 'none') {
+    mobileShow.style.display = 'block';
+  } else {
+    mobileShow.style.display = 'none';
+  }
+});
```

Comment:
- [ ] When I click on each menu item, the menu seems to not close as it is the desired outcome. When I click on the menu, the desired output is it scrolls to the section that is required eg "Projects" takes me to the works section. This works perfectly fine :heavy_check_mark: However, at the same time, it is desired that when I click on the menu item, it closes the menu as it scrolls to the section it is supposed to scroll to.

**This is how your menu works right now:**

[Screencast from 08-06-2023 08:22:15 ALASIRI.webm](https://github.com/sadaf-Daneshgar/First-Project-Microverse/assets/86318284/bb2f0521-2fde-4ce5-8d64-bcd9634e6b2d)

To implement my suggested change, you ca add a uniform class to each menu item eg "lists" and attach an eventlistener that will listen to each individual click and for each close the menu. Below is an example JavaScript code:

```
lists.forEach((list) => {
  list.addEventListener('click', hideMenu);
});
```

In my example above, the code works in assumption that you have a hideMenu function which has code for closing the menu. You can replace the hideMenu with your ```mobileShow.style.display = 'none';``` and it should work perfectly fine as well.